Common/src/linear_solvers_structure.cpp

```diff
@@ -221,14 +232,14 @@ unsigned long CSysSolve::CG_LinSolver(const CSysVector & b, CSysVector & x, CMat
 
   mat_vec(x, A_x);
   r = b; r -= A_x;
-  su2double norm_r = r.norm();
-  su2double norm0 = b.norm();
+  ScalarType norm_r = r.norm();
```

Comment:
You will see some minor changes in the linear solver routines in #652 that introduce a COMM_LEVEL check to decide whether or not we compute residuals and print (save some minor overhead). Just fyi, in case we have conflicts, but they should be very minor